Apps/Src/Task_FaultState.c

```diff
@@ -31,26 +31,27 @@ void EnterFaultState() {
     // Turn LEDs On and logs Error into EEPROM
     BSP_Light_Off(RUN); //turn off run light
     BSP_Light_On(FAULT);
-    SafetyStatus voltStatus = Voltage_CheckStatus();
-    if (voltStatus == OVERVOLTAGE){
-        BSP_Light_On(OVOLT);
-        EEPROM_LogError(FAULT_HIGH_VOLT);
-    }
-    if (voltStatus == UNDERVOLTAGE){
-        BSP_Light_On(UVOLT);
-        EEPROM_LogError(FAULT_LOW_VOLT);
-    }
-    if (Temperature_CheckStatus(Amps_IsCharging()) == DANGER){
-        BSP_Light_On(OTEMP);
-        EEPROM_LogError(FAULT_HIGH_TEMP);
-    }
-    if (Amps_CheckStatus(false) == DANGER){
-        BSP_Light_On(OCURR);
-        EEPROM_LogError(FAULT_HIGH_CURRENT);
-    }
-    if (BSP_WDTimer_DidSystemReset()){
-        BSP_Light_On(WDOG);
-        EEPROM_LogError(FAULT_WATCHDOG);
+    switch (Fault_BitMap){
+        case 1:
+            BSP_Light_On(UVOLT);
+            EEPROM_LogError(FAULT_LOW_VOLT);
+            break;
+        case 2:
+            BSP_Light_On(UVOLT);
+            EEPROM_LogError(FAULT_LOW_VOLT);
+            break;
+        case 4:
+            BSP_Light_On(OTEMP);
+            EEPROM_LogError(FAULT_HIGH_TEMP);
+            break;
+        case 8:
+            BSP_Light_On(OCURR);
+            EEPROM_LogError(FAULT_HIGH_CURRENT);
+            break;
+        default:
+            BSP_Light_On(WDOG); 
+            EEPROM_LogError(FAULT_WATCHDOG);
+            break;
```

Comment:
It is possible to reach this block without faulting because of the watchdog. For example, we could have a hard fault or an open wire

Reply:
But we do not have any log errors defined for those things so I'm not sure what to do for that.

Reply:
Change this so the default isn't the watchdog